x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/geo_field_workspace_panel.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../../mixins';
+
+.lnsVisualizeGeoFieldWorkspacePanel__dragDrop {
+  padding: $euiSizeL;
+  border: $euiBorderThin;
+  border-radius: $euiBorderRadiusSmall;
+
+  &.lnsDragDrop-isDropTarget {
+    @include lnsDroppable;
+    @include lnsDroppableActive;
+
+    p {
+      transition: filter $euiAnimSpeedFast ease-in-out;
+      filter: blur(5px);
+    }
+
+    .lnsExpressionRenderer {
```

Comment:
This and the part below about `lnsDropIllustration__hand` look like a copy-paste leftover